### PR TITLE
allow install hub-addon specify ns

### PR DESCRIPTION
**Pull request by ycyaoxdu**
Signed-off-by: ycyaoxdu <yaoyuchen0626@163.com>

### CODE REVIEW

**Comment by ycyaoxdu** [comment]
related issue #186

File: pkg/cmd/install/hubaddon/exec.go [hunk 1]
```diff
@@ -28,6 +30,15 @@ func (o *Options) complete(cmd *cobra.Command, args []string) (err error) {
 }
 
 func (o *Options) validate() error {
+	kubeClient, err := o.ClusteradmFlags.KubectlFactory.KubernetesClientSet()
+	if err != nil {
+		return err
+	}
+	_, err = kubeClient.CoreV1().Namespaces().Get(context.TODO(), o.values.Namespace, metav1.GetOptions{})
```
Comment (qiujian16) [inline pkg/cmd/install/hubaddon/exec.go:37]: this check is not necessary

**Pull request merged**